docs/pages/virtual.md

```diff
@@ -5,10 +5,13 @@ layout: default
-| [Conda](https://docs.conda.io/projects/conda/en/stable/) | Installs, runs, and updates packages and their dependencies.              | 🟢  |
-| [pipenv](https://pipenv.pypa.io/en/latest/)              | Automatically creates and manages a virtualenv for your projects.         | 🟠  |
-| [pyenv](https://github.com/pyenv/pyenv)                  | Lets you easily switch between multiple versions of Python.               | 🟠  |
-| [virtualenv](https://virtualenv.pypa.io/en/latest/)      | Creates isolated Python environments, and offers more features than venv. | 🟠  |
-| [venv](https://docs.python.org/3/library/venv.html)      | Creates isolated Python environments.                                     | 🔴  |
+| Name                                                              | Short description                                                                                                                                                                         | 🚦  |
+| ----------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: |
+| [conda-forge miniforge](https://github.com/conda-forge/miniforge) | Installs, runs, and updates packages and their dependencies. A trusted sub-set of `conda` with the `conda-forge` channel.                                                                 | 🟢  |
+| [pipenv](https://pipenv.pypa.io/en/latest/)                       | Automatically creates and manages a virtualenv for your projects.                                                                                                                         | 🟠  |
+| [pyenv](https://github.com/pyenv/pyenv)                           | Lets you easily switch between multiple versions of Python.                                                                                                                               | 🟠  |
+| [virtualenv](https://virtualenv.pypa.io/en/latest/)               | Creates isolated Python environments, and offers more features than venv.                                                                                                                 | 🟠  |
+| [venv](https://docs.python.org/3/library/venv.html)               | Creates isolated Python environments.                                                                                                                                                     | 🔴  |
+| [anaconda](https://www.anaconda.com/)                             | Due to recent [licensing ambiguity][anaconda-problems], we recommend avoiding anaconda and many of the default channels. We recommend miniforge or sticking to the `conda-forge` channel. | 🔴  |
+
+[anaconda-problems]: https://www.theregister.com/2024/08/08/anaconda_puts_the_squeeze_on/
```

Comment:
I know I've said this before, I really dislike this syntax. But, I hate inconsistency even more.

Reply:
But then the table becomes huge.

Reply:
But it's fine because `prettier` just does its thing?

Reply:
Maybe I'm just weird in that I read the .md directly. 

Anyway: I pulled them all out. 😛

Reply:
I know 😢 at least it's consistent. Even though the syntax hurts my eyes.

Reply:
Especially if you "read the .md directly", it's like looking at footnotes all the time

Reply:
FWIW I quite like this syntax, and agree it makes the table more readable in its raw markdwon.

Reply:
https://google.github.io/styleguide/docguide/style.html#links

Reply:
That style guide seems to [expcicitly recommend references for long links](https://google.github.io/styleguide/docguide/style.html#reference):

> For long links or image URLs, you may want to split the link use from the link definition

Reply:
That was me admitting defeat, but I still won't use it myself